portfolio/src/main/webapp/gameoflife.js

```diff
@@ -0,0 +1,144 @@
+/* 
+CODE FOR GAME OF LIFE
+    The game of life is a "cellular automota" game, where a grid of cells which can either be "alive" or "dead".
+    Each game step, all the cells are updated based upon the amount live cells around them (either becoming alive, staying alive, or dying).
+    This code contains the logic for the game of life, and displays it in a canvas element. 
+*/
+
+// Animation specific variables
+let canvas = document.getElementById("mycanvas");
+let ctx = canvas.getContext("2d");
+let stop = true;
+let interval, startTime, now, then, elapsed;
```

Comment:
It doesn't look like the "interval" variable is used anywhere. If not, can you remove it?